db/migrate/20230502045343_add_foreign_keys_to_seat.rb

```diff
@@ -0,0 +1,6 @@
+class AddForeignKeysToSeat < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :seats, :ticket, index: true, foreign_key: true
+    add_reference :seats, :trip, index: true, foreign_key: true
```

Comment:
rollback for this migration works?

Reply:
Works for both the files vhaia.

![Screenshot 2023-05-15 at 2 48 55 PM](https://github.com/hoh-wtag/bus_ticketing_system/assets/130047353/4f962083-6de7-4a05-9d5d-c6222a75f4d9)